### PR TITLE
 Add loc/submitted-on indices for daily-feeding UCR

### PR DESCRIPTION
**Pull request by sravfeyn**

### CODE REVIEW

File: custom/icds_reports/ucr/data_sources/daily_feeding_forms.json [hunk 1]
```diff
@@ -960,11 +960,30 @@
     },
     "engine_id": "icds-ucr",
     "sql_column_indexes": [
+      // primarily used for MPR report
```
Comment (emord) [inline custom/icds_reports/ucr/data_sources/daily_feeding_forms.json]: json doesn't have coments

**Comment by sravfeyn** [comment]
Adding a check list to keep track of progress
- [ ] Finish index creation on india
- [ ] Finish index creation on ICDS
- [ ] Merge this
- [ ] Delete old indices

File: custom/icds_reports/ucr/data_sources/daily_feeding_forms.json [hunk 2]
```diff
@@ -960,11 +960,30 @@
...
       {
         "column_ids": [
```
Comment (sravfeyn) [inline custom/icds_reports/ucr/data_sources/daily_feeding_forms.json]: For some reason, this index doesn't seem to exist on India.

File: custom/icds_reports/ucr/data_sources/daily_feeding_forms.json [hunk 3]
```diff
@@ -960,11 +960,30 @@
...
           "awc_id",
```
Comment (sravfeyn) [inline custom/icds_reports/ucr/data_sources/daily_feeding_forms.json]: I found below index on ICDS for this.

```indexname | ix_daily_feeding_forms_awc_submitted_on_multi
indexdef  | CREATE INDEX ix_daily_feeding_forms_awc_submitted_on_multi ON public."ucr_icds-cas_static-daily_feeding_forms_85b1167f" USING btree (awc_id, submitted_on)```

Comment (sravfeyn) [reply custom/icds_reports/ucr/data_sources/daily_feeding_forms.json]: I am creating the indices anyway, since they can be renamed if required.

**Comment by sravfeyn** [comment]
Index creation is in progress on ICDS (only for icds-cas domain UCR tables) for this and the other two PRs (https://github.com/dimagi/commcare-hq/pull/23883/, https://github.com/dimagi/commcare-hq/pull/23919).

**Comment by sravfeyn** [comment]
Adding a check list to keep track of progress

- [ ] Finish index creation on ICDS
- [ ]  Merge this
- [ ]  Delete old indices

**Comment by sravfeyn** [comment]
@emord

**Pull request merged**